lib/active_admin/router.rb

```diff
@@ -65,10 +65,10 @@ def resource_routes(config)
 
     def page_routes(config)
       page = config.underscored_resource_name
-      router.get "/#{page}" => "#{page}#index"
+      router.get "/#{page}", to: "#{page}#index"
       config.page_actions.each do |action|
         Array.wrap(action.http_verb).each do |verb|
-          build_route(verb, "/#{page}/#{action.name}" => "#{page}##{action.name}")
+          build_route(verb, "/#{page}/#{action.name}", to: "#{page}##{action.name}")
```

Comment:
I prefer we do this change in a separate PR to close out the issue since this is not dependent on Rails 8 tests. This route signature is already long supported and thus is backwards compatible.

Reply:
@javierjulio yes, I've committed here just for testing and to make sure that there aren't other deprecations, then I will backport to a separate PR

Reply:
@tagliala ok great, thank you.